tests/test_raster.py

```diff
@@ -1381,6 +1381,13 @@ def test_reproject(self, example: str) -> None:
         assert r_test.crs.to_epsg() == 4326
 
         # -- Additional tests --
+        # First, make sure dst_bounds extend beyond current extent to create nodata
```

Comment:
So this is a replacement of the test never run? Should we delete that test that is never run then?

Reply:
Sorry, there's a mistake I think. `r_test` should be used in the following lines, because I made sure it contains nodata. Before these lines were added, there was actually no gap so the test was pointless...
I'll check again!

Reply:
In the end, no it was correct, `r_test` is just used to check that reproject with these bounds will creates nodata in the following tests, not used directly in those tests.
I had to follow several layers of git blame to track down the origin of this test. It was created [here](https://github.com/GlacioHack/geoutils/pull/223), but then modified a few times, particularly [here](https://github.com/GlacioHack/geoutils/pull/284) by you @rhugonnet 😜, which made the test lose its usefulness...
In short, it was just added at a point where rasterio's default nodata values would add very large values. I'm not sure this test is super useful nowadays, but I will keep it just in case we break something very badly.